src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
-body {
+$day-size: 100px;
+$week: 7;
+$gap: 1px;
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  display: flex;
+  width: 100%;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+  width: 707px;
+  box-sizing: content-box;
```

Comment:
The `box-sizing` property for `.calendar` should be set to `border-box` to follow the code style rules and ensure that the padding and border are included in the element's total width and height.